Source/Core/DiscIO/CompressedBlob.cpp

```diff
@@ -154,6 +154,22 @@ bool CompressFileToBlob(const std::string& infile, const std::string& outfile, u
 		return false;
 	}
 
+	File::IOFile inf(infile, "rb");
+	if (!inf)
+	{
+		PanicAlertT("Failed to open the input file %s.", infile.c_str());
```

Comment:
I'm a fan of quotes around string parameters that denote paths or filenames. Always looks odd when a message pops up "Failed to open the input file ." for an empty string; while "Failed to open the input file ''." makes it a bit easier to notice that it didn't mean the dot.